<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>labelprint_goodinfo_dialog</class>
 <widget class="QWidget" name="labelprint_goodinfo_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,0">
     <item>
      <widget class="uinormal_button" name="pushButton_goodadd">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="uinormal_button" name="pushButton_goodremove">
       <property name="text">
        <string>移除</string>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font:10pt;
color:red;</string>
       </property>
       <property name="text">
        <string>表格选中某单元后，单击可以修改内容</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>uinormal_button</class>
   <extends>QPushButton</extends>
   <header>uinormal_button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
